src/main/java/net/dries007/tfc/common/tileentity/AbstractFirepitTileEntity.java

```diff
@@ -56,6 +57,42 @@ public static void convertTo(IWorld world, BlockPos pos, BlockState state, Abstr
         }
     }
 
+    public static Triple<Integer, Float, Long> consumeFuelForTicks(long deltaPlayerTicks, IItemHandlerModifiable inventory, int burnTicks, float burnTemperature, int slotStart, int slotEnd)
```

Comment:
This is gross but w/e. Although probably could move to `Helpers` with our other inventory helper functions.